LICENSE

```diff
@@ -0,0 +1,1170 @@
+The Open Computing Facility's directors and members believe strongly in
+the importance of free and open-source software, and we hope for
+software and other works developed by our organization to be used to
+advance the free software movement.
+
+We have spent a considerable amount of time discussing which licenses
+best reflect our organization's mission, with a roughly even split among
+members who believe that a copyleft license like the GPL best promotes
```

Comment:
There are a few instances of the bare term "GPL" which should be replaced with "GNU GPL" or "GPLv3". "GPL" stands for "General Public License" and, without any qualifier, doesn't technically specify any license. This isn't an issue in informal conversation but we should be as clear as possible in our license.